### ISSUE TITLE
[MAS1.4.11] [Visual Requirements-Settings] Luminosity Ratio is less than 3:1 for border outline with respective background colour for checkboxes 

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1771]
**Actual Result:**
Luminosity Ratio is less than 3:1 for border outline with respective background colour for checkboxes 
“User Settings ,Application updates ,Data collection” sections under checkboxes 

**Expected Result:**
Luminosity Ratio should be greater than or equal to 3:1 for border outline with respective background colour for checkboxes 

**Note:** 
This issue exits for throughout application for all checkboxes and button controls 
Same issue observed with Mac Os.

**Repro Steps:**
1.Open the Bot Framework V4 Emulator.
2.Navigate to Welcome tab screen and select it, screen gets opened  
3.Navigate to the open bot and select it  
4.Open bot dialog gets opened and connect to the open bot 
5.Live chat screen gets appears 
6.Navigate to the Edit ngrok settings and select it 
7.Emulator screen gets opened 
8.Navigate to the “1. User settings 2. Application updates 3. Data collection” section check boxes 
9. Open 'Colour Contrast Analyser' application.
10.Observe the issue

**User Impact:**
Users with low vision or colour blindness needs that difference between contrast of text is always high.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Browser: Edge
Application: Bot Framework V4 Emulator
Tool: Colour contrast Analyzer

OS: MacOs Version 10.15
Application: Bot Framework V4 Emulator (Version 4.5.2-80829)

**MAS Reference Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B6CA3BF14-2635-434A-832C-28DD9A18B9FC%7D&file=MAS%201.4.11%20%E2%80%93%20Non-text%20Contrast.docx&action=default&mobileredirect=true&cid=c3837dea-3bb1-4c5a-9374-19ec384f14c6](url)
![MAS1 4 11_Settings](https://user-images.githubusercontent.com/53518994/63512408-66180480-c501-11e9-9392-d20daeba2036.png)

![Cancel button border- MAC](https://user-images.githubusercontent.com/53518994/66312145-b28e9680-e92d-11e9-8d8e-c1e705ec515f.png)

**Comment by corinagum** [comment]
@Kumar2608, @awalia13 could we get clarification on this issue? 
Focus for checkboxes have been verified as fixed in #1712. For that part of this bug, I believe there is no action item. Can we confirm?

Regarding the last screenshot above, it looks like the buttons at the bottom of the dialog should be a separately filed bug. What does "highlighted control border" mean? Does that mean mouseover/hover?

Cancel button (no mouseover)
![image](https://user-images.githubusercontent.com/14900841/66687289-0c95a180-ec37-11e9-9b8f-7e5d4f912be7.png)

Cancel button (mouseover)
![image](https://user-images.githubusercontent.com/14900841/66687302-17503680-ec37-11e9-886a-fb227491acae.png)

Should one or both of these Cancel button states be modified?

The button next to it, Connect, is disabled in these screenshots. Does this also need to have a higher contrast value when disabled?

Thanks for the clarification!

**Comment by ShikhaMishra11** [comment]
Above "highlighted control border" means control inside that Red border which is "Cancel button" and as per new WCAG2.1 boundary and background of the control should be greater than 3.1 
and also disabled controls are not comes under WCAG2.1.

**Comment by ShikhaMishra11** [comment]
Issue got fixed so closing the bug.
